integration/src/test/scala/org/knora/webapi/E2ESpec.scala

```diff
@@ -119,7 +119,9 @@ abstract class E2ESpec
     Unsafe.unsafe(implicit u => runtime.unsafe.shutdown())
 
   protected def singleAwaitingRequest(request: HttpRequest): HttpResponse =
-    UnsafeZioRun.runOrThrow(ZIO.serviceWithZIO[TestClientService](_.singleAwaitingRequest(request)))
+    UnsafeZioRun.runOrThrow(
+      ZIO.serviceWithZIO[TestClientService](_.singleAwaitingRequest(request, printFailure = true))
+    )
```

Comment:
as discussed, you wanted to hide this parameter behind a default

Reply:
thx for the heads up, done in https://github.com/dasch-swiss/dsp-api/pull/3043/commits/bc51e29b4b065eb4a3c2afdd7de2a604ac032c73